### PR TITLE
Add user SSH on job submission page. Generate SSH key pair on rest se…

### PR DESCRIPTION
**Pull request by wangdian**
Add user SSH on job submission page.
Generate SSH key pair on rest server

### CODE REVIEW

File: src/webportal/src/app/job-submission/utils/constants.js [hunk 1]
```diff
@@ -144,3 +144,16 @@ export const DOCKER_OPTIONS = [
   },
 ];
 export const DEFAULT_DOCKER_URI = 'openpai/tensorflow-py36-cu90';
+// For PAI runtime only
+export const PAI_PLUGIN = 'com.microsoft.pai.runtimeplugin';
```
Comment (sunqinzheng) [inline src/webportal/src/app/job-submission/utils/constants.js]: I'm not sure if task's ssh connection will be a feature that could only be accessed by web portal.
If not, I think we should add some documents about job protocol's extra field format and move some logics to rest server.

**Comment by scarlett2018** [comment]
**TODO**: add document for usage instruction

File: src/webportal/src/app/env.js.template [hunk 2]
```diff
@@ -1,4 +1,5 @@
 window.ENV = {
+  restServerLauncherType: '${REST_SERVER_LAUNCHER_TYPE}',
```
Comment (sunqinzheng) [inline src/webportal/src/app/env.js.template]: I have added a environment variable `launcherType` and it is already checked in. Please used that instead.

**Comment by wangdian** [comment]
Separate this to multiple PRs, close this.
Please refer to #3596

**Pull request closed without merging**